README.md

```diff
@@ -1,30 +1,33 @@
+**Session Abstract:** In this workshop you will be introduced to the [Microsoft Azure Cognitive Services](https://azure.microsoft.com/en-gb/services/cognitive-services/?WT.mc_id=ainights-github-amynic), a range of offerings you can use to infuse intelligence and machine learning into your applications without needing to build the code from scratch.
 We will cover pre-trained AI APIs, such as [computer vision](https://azure.microsoft.com/en-gb/services/cognitive-services/directory/vision/?WT.mc_id=ainights-github-amynic) and [text analytics](https://azure.microsoft.com/en-gb/services/cognitive-services/directory/lang/?WT.mc_id=ainights-github-amynic), that are accessed by REST protocol. nEXT we will dive into Custom AI that uses transfer learning - [Microsoft Azure Custom Vision](https://azure.microsoft.com/en-gb/services/cognitive-services/custom-vision-service/?WT.mc_id=ainights-github-amynic). This enables you to provide a small amount of your own data to train an image classification model. Wrapping the workshop up by building our custom trained AI into an application - using [Logic Apps](https://azure.microsoft.com/en-gb/services/logic-apps/?WT.mc_id=ainights-github-amynic), this technology is ideal for building data pipeline processes that work with your machine learning models.
 
-
 **PowerPoint Slides available here:** [Creating applications that can see, hear, speak or understand - using Microsoft Cognitive Services](ai-nights-beginner.pptx)
 
 > When you see **[BONUS SECTION]** in this content it means it is not designed for the 1 hour slot for the Cognitive Services within the beginner track, however if you wish to customise your session or learn about other parts of the Cognitive services you can us this content
 
-
 ## Pre-requisites for your machine
+
 * Clone this repository to your local machine to gain images and code samples you need for the demos: ```git clone https://github.com/amynic/ainights-sessionowners.git```
+If you plan to clone other Global AI Night workshops in the same folder, please use the following command to distinguish them since most have similar root directory names
+```git clone https://github.com/amynic/ainights-sessionowners.git BeginnerCognitiveServices```
 * [Microsoft Azure Subscription](https://azure.microsoft.com/en-gb/free/?WT.mc_id=ainights-github-amynic)
 * Laptop with a modern web browser (Google Chrome, Microsoft Edge)
-* [Postman, API Development Environment - available on Windows, Linux and macOS](https://www.getpostman.com/downloads/)
+* [Postman, API Development Environment - available on Windows, Linux and macOS](https://www.getpostman.com/downloads/) To ensure WiFi at the venue is not throttled, attendees can be informed to have this pre installed before arriving at the venue
```

Comment:
Please can highlight that this is BONUS content and not what was shown in train-the-rainer and is not needed unless people want to go the extra mile

Reply:
Yup this will be changed

Reply:
In the one hours session worth of content I recommend you demo Postman as the speaker but the attendees will only use custom vision (web browser) and logic apps (web browser)

Reply:
Postman is just not used in the Text Analytics but also in the Logic Apps Demo too where, IMO its a valuable gatekeeping/sanity-check of validating the Prediction URL generated by Custom Vision and only then progressing to use that validated URL in the step function of the Logic Apps.

Reply:
OK - good feedback thank you